### DISCUSSION TITLE
Read-Only Mode for CardDAV

### DISCUSSION THREAD

**Discussion by Guenni99** [ckulka/baikal-docker#102] [Ideas]
Hello Community,

im new here and im using the "sabredav - Baikal Docker Image" to share addressbooks with a few iOS Devices.
This works really fine so far.
But my Question is, is there an Opportunity to set addressbooks or principals as read-only?

At the Moment the iOS User can write directly into my Database.
I have seen that there are some features like an ACL Plugin but i have no clue how to use it.

Thank you really much!

Paul

**Comment by BobWs** [comment]
If you only let users subscribe then they will have only read access.

**Reply by Guenni99** [reply]
Hello BobWs, 
I think i dont understand.
In my Case i share the CardDAV URL's via Mobile Device Management. How can users only subscribe? 

Thank you in advance
Paul

**Reply by BobWs** [reply]
> Hello BobWs, I think i dont understand. In my Case i share the CardDAV URL's via Mobile Device Management. How can users only subscribe?
> 
> Thank you in advance Paul

never mind, sorry I tested my theory but it's working!

**Reply by Guenni99** [reply]
could you please be so kind and share your theory with me?
I dont know how to set it up that the iOS devices only have read access to my CardDAV.

**Reply by BobWs** [reply]
> could you please be so kind and share your theory with me? I dont know how to set it up that the iOS devices only have read access to my CardDAV.

Sorry I made a typo, I meant it doesn't work!

**Comment by ckulka** [comment]
Hi @Guenni99 , I think this question is best asked over at the [sabre-io/Baikal](https://github.com/sabre-io/Baikal) repository.

This repository just provides the Docker image for Baikal. Your question is around Baikal functionality, which they should be able to answer... at least better than me, as I don't know the code well enough.

**Comment by cgrohan** [comment]
Hi @Guenni99, were you successful? Thx
